frontend/apps/reader/modules/readerlink.lua

```diff
@@ -135,6 +139,9 @@ function ReaderLink:addToMainMenu(menu_items)
                     G_reader_settings:saveSetting("swipe_to_jump_to_latest_bookmark",
                         not isSwipeToJumpToLatestBookmarkEnabled())
                 end,
+                help_text = _([[Swipe to the left to go the most recently bookmarked page.
+This can be useful to quickly swipe back and forth between your reading and some reference page (for example a map, a characters list...)
```

Comment:
Maybe…

`between what you are reading and some reference page (for example, notes, a map or a character list).